setup.py

```diff
@@ -31,6 +31,7 @@
 
 npm_packages_path = "./dist/*.tgz"
 auto_extension_path = "./etc/config/jupyter_notebook_config.d/*.json"
+notebook_config_path = "./etc/config/*.py"
```

Comment:
Could use just use a .json file (I suspect it would be `nbresuse.json`) and include it in `auto_extension_path` along with the others?  This would be much preferred over this approach.  Isn't something like this needed to indicate the server extension is enabled anyway?

I would expect it to look something like:
```json
{
  "NotebookApp": {
    "nbserver_extensions": {
      "nbresuse": true
    },
    "ResourceUseDisplay": {
      "track_cpu_percent": true
    }
  }
}
```

Reply:
I tried using a json file like this, but I wasn't able to get it to work